app/screens/post_options/post_options.tsx

```diff
@@ -73,64 +81,85 @@ const PostOptions = ({
-                    />
+                <MarkAsUnreadOption
+                    post={post}
+                    sourceScreen={sourceScreen}
+                />
                 }
                 {canCopyPermalink &&
-                    <CopyPermalinkOption
-                        post={post}
-                        sourceScreen={sourceScreen}
-                    />
+                <CopyPermalinkOption
+                    post={post}
+                    sourceScreen={sourceScreen}
+                />
                 }
                 {!isSystemPost &&
-                    <SaveOption
-                        isSaved={isSaved}
-                        postId={post.id}
-                    />
+                <SaveOption
+                    isSaved={isSaved}
+                    postId={post.id}
+                />
                 }
                 {Boolean(canCopyText && post.message) &&
-                    <CopyTextOption
-                        postMessage={post.message}
-                        sourceScreen={sourceScreen}
-                    />}
+                <CopyTextOption
+                    postMessage={post.message}
+                    sourceScreen={sourceScreen}
+                />}
                 {canPin &&
-                    <PinChannelOption
-                        isPostPinned={post.isPinned}
-                        postId={post.id}
-                    />
+                <PinChannelOption
+                    isPostPinned={post.isPinned}
+                    postId={post.id}
+                />
                 }
                 {canEdit &&
-                    <EditOption
-                        post={post}
-                        canDelete={canDelete}
-                    />
+                <EditOption
+                    post={post}
+                    canDelete={canDelete}
+                />
                 }
                 {canDelete &&
                 <DeletePostOption
                     combinedPost={combinedPost}
                     post={post}
                 />}
+                {shouldShowBindings &&
+                <AppBindingsPostOptions
+                    post={post}
+                    serverUrl={serverUrl}
+                    bindings={bindings}
+                />
+                }
             </>
         );
     };
 
     const additionalSnapPoints = 2;
+    const lowerSnapPoints = snapPoints + additionalSnapPoints;
+
+    let upperSnapPoints = lowerSnapPoints;
+    if (shouldShowBindings) {
+        upperSnapPoints += bindings.length;
+    }
 
     return (
         <BottomSheet
             renderContent={renderContent}
             closeButtonId={POST_OPTIONS_BUTTON}
             componentId={Screens.POST_OPTIONS}
-            initialSnapIndex={0}
-            snapPoints={[((snapPoints + additionalSnapPoints) * ITEM_HEIGHT), 10]}
+            initialSnapIndex={1}
+            snapPoints={[upperSnapPoints * ITEM_HEIGHT, lowerSnapPoints * ITEM_HEIGHT, 10]}
```

Comment:
I don't see the reason of having three snap points when there are no bindings..

in short, this should change if there are app bindings, if not it should remain the exact way it was before this PR.